### ISSUE TITLE
Version 3.0.0 publishing

### ISSUE THREAD

**Issue by alex-vasenin** [onekiloparsec/SwiftAA#123]
Is it published or what? I see it merged into master, but both CocoaPods and SPM list 2.4.0 as the latest.

**Comment by alex-vasenin** [comment]
The reason I'm asking is that I want to migrate my `RiseTransitSet2` wrapper to either ObjC or C, but I don't know which one.

**Comment by tallPete** [comment]
I would love to see this too, especially to see the newer RiseTransitSet2 wrapper, but I understand that it takes a lot of work to do the release I believe. Perhaps the workload could be cut down by only supporting SPM for instance going forward. I also know that as soon as a new version is posted there will be issue after issue posted of "my code doesn't work any more", which onekiloparsec may not have time or more likely the will to handle. So I can appreciate holding the current position. I'd help out myself but I don't know enough about how to release a new version.

**Comment by alex-vasenin** [comment]
> especially to see the newer RiseTransitSet2 wrapper

Frankly, I'm somewhat disappointed in the `RiseTransitSet2` implementation in AA+. Original `RiseTransitSet` is based on Meeus's book and it's fast and elegant (with a few known caveats), while `RiseTransitSet2` uses a brute-force approach simply iterating over time with 10-minute steps. I even found a bug in it (it's fixed in the latest version).

**Comment by Yann-Vaonis** [comment]
+1 for this, it would be nice to have the 3.0 version tagged so that it can be download with SPM.

**Comment by onekiloparsec** [comment]
It's been so long time. I don't know how to do it again... I need to check if my Xcode installation is up to date. Has the process evolved? Can anyone provide some guidance? SPM is the Swift package manager, right? But we should also publish in Cocoapods.

**Comment by onekiloparsec** [comment]
And.. that's it? That's very easy. @alex-vasenin ok for you to tag SwiftAA as it is right now?

Side question @Yann-Vaonis : quite logically, you seem to work for Vaonis, with whom I am in contact since quite some time! If that really the case, I am very happy to know that SwiftAA is being used in Vaonis app, you confirm?

**Comment by Yann-Vaonis** [comment]
> Side question [@Yann-Vaonis](https://github.com/Yann-Vaonis) : quite logically, you seem to work for Vaonis, with whom I am in contact since quite some time! If that really the case, I am very happy to know that SwiftAA is being used in Vaonis app, you confirm?

Yes, I can confirm, we're using SwiftAA in the Singularity and Gravity apps :)

**Comment by alex-vasenin** [comment]
Sorry guys, I can't help you now. I'm just started a new job last week and I'm ill.
